org.eclipse.scout.rt.app/src/main/java/org/eclipse/scout/rt/app/ApplicationProperties.java

```diff
@@ -358,4 +360,157 @@ public String description() {
       return "Specifies whether the application context handler should also lookup resources w/o base resource. The default value is false.";
     }
   }
+
+  public static class ScoutApplicationGzipEnabled extends AbstractBooleanConfigProperty {
+
+    @Override
+    public String getKey() {
+      return "scout.app.gzip.enabled";
+    }
+
+    @Override
+    public String description() {
+      return "Specifies whether " + GzipHandler.class.getName() + " is used.";
```

Comment:
Add "Default value is " + getDefaultValue();"

Reply:
https://github.com/eclipse-scout/scout.rt/compare/ddfce395c1ce7e5e0bb3ec14ca3342013320f395..3e80c6ba06d95d99f3212f4e027768c687cc2451